alpha/hal.cpp

```diff
@@ -304,6 +307,16 @@ void _TS_HAL::power_set_mic(bool enabled)
   EXIT_CRITICAL;
 }
 
+uint8_t _TS_HAL::power_get_batt_level(){
+  long level;
+  ENTER_CRITICAL;
+#ifdef HAL_M5STICK_C
+  level = constrain(map(M5.Axp.GetVbatData() * 1.1, 3100, 4000, 0, 100), 0, 100);
```

Comment:
put the constrain call outside the critical section

Reply:
Fixed!